### PR TITLE
WEC-Sim QTFs compatability with NEMOH

### PR DESCRIPTION
**Pull request by MShabara**
This PR adds the three QTF features:
1- Reading the NEMOH QTF outputs to BIMIO.
2- Does the contour plots of the mod of the sum and diff.
3- Adds a BIMIO/NEMOH example for a FOWT subjected to QTFs.

With this update WEC-Sim should be able to compute the second order forces based on the NEMOH outputs.

### CODE REVIEW

**Review by akeeste — CHANGES_REQUESTED** [review]
Thanks @MShabara. I have a few questions and suggestions on this PR before merging, summarized here:
- revert the default test behavior to running all tests
- updates to plotBEMIO and plotQTFs: these changes will make plotQTFs more robust. In my testing they worked for inputs ``plotBEMIO(hydro1)``, ``plotQTFs(hydro1)``, ``plotBEMIO(hydro1, hydro2)``, ``plotQTFs(hydro1, hydro2``)
- Confirm that changes to ``assignIDs``, and ``triToFullMatrix`` still function with readWAMIT
- suggestion to make column indexing more robust in ``triToFullMatrix``

File: source/functions/BEMIO/QTFs/assignIDs.m [hunk 1]
```diff
@@ -26,8 +26,8 @@
     %     input table for further processing.
     %
 
-    vector_j = dof_data(:,1);
-    vector_i = dof_data(:,2);
+    vector_i = dof_data(:,1);
+    vector_j = dof_data(:,2);
```
[the remark this answers is not in the document]
Comment (MShabara) [reply source/functions/BEMIO/QTFs/assignIDs.m:30]: This just enhances the code readability, this matrix is symmetric and should no effect on WAMIT results.

**Comment by MShabara** [comment]
Hi @akeeste 

I responded to your question, and approved all the commits suggested. Please take a look and let me know what you think.

**Comment by akeeste** [comment]
Thanks @MShabara. The tests are passing again, merging...

**Pull request merged**